src/mono/wasm/wasm.proj

```diff
@@ -67,21 +70,100 @@
       OutputFileName="$(TimeZonesDataPath)" />
   </Target>
 
+  <Target Name="GenerateEmccPropsAndRspFiles">
+    <ItemGroup>
+      <_EmccCommonFlags Include="-s ALLOW_MEMORY_GROWTH=1" />
+      <_EmccCommonFlags Include="-s NO_EXIT_RUNTIME=1" />
+      <_EmccCommonFlags Include="-s FORCE_FILESYSTEM=1" />
+      <_EmccCommonFlags Include="-s &quot;EXTRA_EXPORTED_RUNTIME_METHODS=['ccall', 'FS_createPath', 'FS_createDataFile', 'cwrap', 'setValue', 'getValue', 'UTF8ToString', 'UTF8ArrayToString', 'addFunction']&quot;" />
+      <_EmccCommonFlags Include="-s &quot;EXPORTED_FUNCTIONS=['_putchar']&quot;" />
+      <_EmccCommonFlags Include="--source-map-base http://example.com" />
+      <_EmccCommonFlags Include="-emit-llvm" />
+
+      <_EmccCommonFlags Include="-s MODULARIZE=1" Condition="'$(WasmEnableES6)' != 'false'" />
+      <_EmccCommonFlags Include="-s EXPORT_ES6=1" Condition="'$(WasmEnableES6)' != 'false'" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(OS)' != 'Windows_NT'">
+      <_EmccFlags Include="--profiling-funcs" />
+      <_EmccFlags Include="@(_EmccCommonFlags)" />
+
+      <_EmccFlags Include="-DENABLE_METADATA_UPDATE=1" Condition="'$(WasmEnableMetadataUpdate)' == 'true'" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+      <_EmccFlags Include="@(_EmccCommonFlags)" />
+
+      <_EmccFlags Include="-s WASM=1" />
+      <_EmccFlags Include="-s BINARYEN=1" />
+      <_EmccFlags Include="-s ALIASING_FUNCTION_POINTERS=0" />
+      <_EmccFlags Include="-s USE_ZLIB=1" />
+      <_EmccFlags Include="-DENABLE_NETCORE=1" Condition="'$(Configuration)' == 'Release'" />
```

Comment:
```suggestion
```
I don't think this is used anymore?  Also I'm not sure why it would be conditioned on the config either way.